openedx_tagging/core/tagging/rules.py

```diff
@@ -99,6 +99,16 @@ def can_view_object_tag_objectid(_user: UserType, _object_id: str) -> bool:
     return True
 
 
+@rules.predicate
+def can_remove_object_tag_objectid(_user: UserType, _object_id: str) -> bool:
+    """
+    Everybody can remove object tags from any objects.
+
+    This rule could be defined in other apps for proper permission checking.
+    """
+    return True
```

Comment:
Can you move this down below `can_change_object_tag_objectid` ? Also the default value shouldn't be True - the default value should be the same as `can_change_object_tag_objectid`, so just call that function and return the same value.

Reply:
@bradenmacdonald Updated here https://github.com/openedx/openedx-learning/pull/188/commits/bf49f694ece7c1bcc0fc50b8919111ecb1b158eb